cheat.php

```diff
@@ -164,14 +164,16 @@
 	if( isset( $Data[ 'response' ][ 'new_score' ] ) )
 	{
 		$Data = $Data[ 'response' ];
+		
+		$ScoreTable = array(0, 1200, 2400, 4800, 12000, 30000, 72000, 180000, 450000, 1200000, 2400000, 3600000, 4800000, 6000000);
 
 		echo PHP_EOL;
 
 		Msg(
 			'>> Your Score: {lightred}' . number_format( $Data[ 'new_score' ] ) .
 			'{yellow} (+' . number_format( $Data[ 'new_score' ] - $Data[ 'old_score' ] ) . ')' .
 			'{normal} - Current level: {green}' . $Data[ 'new_level' ] .
-			'{normal} (' . number_format( $Data[ 'new_score' ] / $Data[ 'next_level_score' ] * 100, 2 ) . '%)'
+			'{normal} (' . number_format( ($Data[ 'new_score' ] - $ScoreTable[ $Data[ 'new_level' ] - 1]) / ($Data[ 'next_level_score' ] - $ScoreTable[ $Data[ 'new_level' ] - 1 ] ) * 100, 2 ) . '%)'
```

Comment:
Need an isset check if there's no xp data

Reply:
I don't know how to do that. Also, it works up to lvl 15 so it will be a while b4 someone reaches level 16

Reply:
Guess you should read up xD https://secure.php.net/manual/en/function.isset.php